### PR TITLE
For #1425: MkHook implementation

### PR DESCRIPTION
**Pull request by paulodamaso**
For #1425:
- Added `MkHookTest` to test `MkHook`
- Left puzzle to complete implementation of `MkHook`

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#1428](https://github.com/jcabi/jcabi-github/issues/1428) is now in scope, role is `REV`

**Comment by llorllale** [comment]
@paulodamaso I think it's a good idea if you go back to the original issue's description and include the name of the method that's throwing the `UnsupportedOperationException`

File: src/test/java/com/jcabi/github/mock/MkHookTest.java [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+ */
+public final class MkHookTest {
+
+    /**
+     * Test if {@link MkHook} is being created properly.
+     *
+     * @throws Exception If something goes wrong
+     */
+    @Test(expected = UnsupportedOperationException.class)
+    public void create() throws Exception {
```
Comment (llorllale) [inline src/test/java/com/jcabi/github/mock/MkHookTest.java]: @paulodamaso remember - one assertion per test case. So, please split this test into several, each with a single test case

File: src/test/java/com/jcabi/github/mock/MkHookTest.java [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+ *
+ * @author Paulo Lobo (pauloeduardolobo@gmail.com)
+ * @version $Id$
+ * @since 0.42
+ */
+public final class MkHookTest {
+
+    /**
+     * Test if {@link MkHook} is being created properly.
+     *
```
Comment (llorllale) [inline src/test/java/com/jcabi/github/mock/MkHookTest.java]: @paulodamaso nitpick: please remove the blank line between the method's descriptive javadoc and the tags.

**Comment by paulodamaso** [comment]
@llorllale Done, please take a look.

**Comment by paulodamaso** [comment]
@llorllale ping

**Comment by 0crat** [comment]
@llorllale[/z](https://www.0crat.com/u/llorllale) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by paulodamaso** [comment]
@llorllale please give you veredict about this PR

File: src/test/java/com/jcabi/github/mock/MkHookTest.java [hunk 3]
```diff
@@ -0,0 +1,163 @@
+/**
+ * Copyright (c) 2013-2017, jcabi.com
```
Comment (amihaiemil) [inline src/test/java/com/jcabi/github/mock/MkHookTest.java]: @paulodamaso Just a small correction, ``2013-2018`` here

**Comment by paulodamaso** [comment]
@amihaiemil Great, corrected, please take a look. For info, there are a lot of files with outdated copyright info, I've filed an issue (https://github.com/jcabi/jcabi-github/issues/1429).

**Comment by amihaiemil** [comment]
@paulodamaso hmm, strange. Now Qulice complains about the header:

```
[ERROR] /src/test/java/com/jcabi/github/mock/MkHookTest.java[2]: Line does not match expected header line of ' * Copyright (c) 2013-2017, jcabi.com'. (HeaderCheck)
```

I'd say undo the change (leave it 2017) and it will be covered in the ticket. Probably Qulice takes the years from the LICENCE file as reference and that's why it's complaining.

**Comment by amihaiemil** [comment]
@rultor merge it

**Comment by 0crat** [comment]
Order was finished: +15 point(s) just awarded to @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by 0crat** [comment]
The job [#1428](https://github.com/jcabi/jcabi-github/issues/1428) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @amihaiemil[/z](https://www.0crat.com/u/amihaiemil)

**Pull request merged**